Source/Core/Core/IOS/IOS.h

```diff
@@ -45,6 +46,38 @@ struct IPCReply
   u64 reply_delay_ticks;
 };
 
+// Used to make it more convenient for functions to return timing information
+// without having to explicitly keep track of ticks in callers.
+class Ticks
+{
+public:
+  Ticks(u64* ticks = nullptr) : m_ticks(ticks) {}
```

Comment:
`explicit` ?

Reply:
In this case, we actually want implicit conversions to Ticks as this class is for all intents and purposes just a wrapper around a u64* and having to write Ticks{...} every time would be very verbose.